src/frontend/admin/src/components/presentational/files/thumbnail/FileThumbnail.tsx

```diff
@@ -1,15 +1,24 @@
 import * as React from "react";
+import { ReactNode } from "react";
 import Box from "@mui/material/Box";
 import IconButton from "@mui/material/IconButton";
 import Typography from "@mui/material/Typography";
-import { Close, FileOpen } from "@mui/icons-material";
+import ChangeCircleOutlinedIcon from "@mui/icons-material/ChangeCircleOutlined";
+import { ThumbnailDetailField } from "@/services/api/models/Image";
 
 interface Props {
-  file: File;
-  onDelete?: (file: File) => void;
+  file: File | ThumbnailDetailField;
+  onReplace?: () => void;
 }
 
+const isThumbnail = (
+  item: File | ThumbnailDetailField,
+): item is ThumbnailDetailField => {
+  return "filename" in item;
+};
+
 export function FileThumbnail({ file, ...props }: Props) {
+  const isThumb = isThumbnail(file);
```

Comment:
<img width="794" alt="Capture d’écran 2023-12-14 à 17 03 37" src="https://github.com/openfun/joanie/assets/9628870/d7e220c7-9d0a-4863-817f-e6742a540370">

When the text overflows the left padding changes, it's not a big deal but I noticed it